src/software/world/robot.h

```diff
@@ -9,6 +9,7 @@
 #include "software/util/time/timestamp.h"
 #include "software/world/robot_capabilities.h"
 
+using RobotId = unsigned int;
```

Comment:
We should use this throughout the `Robot` class as well eh? Totally your call about whether or not that's in-scope for this PR, just please file an issue if you don't think it is.

Reply:
done